src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -1141,14 +1165,17 @@ public class OpcodeLogicToBool : Opcode
 
         public override void Execute(ICpu cpu)
         {
+            // This may look like it's just pointlessly converting from a
+            // ScalarBoolean to a primitive boolean and then back into a
+            // ScalarBoolean, and in the case where the operand was already
+            // a ScalarBoolean that would be true.  But the purpose of this opcode
+            // is to also change integers and floats into booleans. Thus the call to
+            // Convert.ToBoolean():
```

Comment:
Hmm. if it already works lets not omit the feature after all.

By the way, the OpcodeLogicToBool seems to be only used in one place, the `toggle` command.